qos/qos_rule.go

```diff
@@ -309,7 +291,7 @@ func (ipf *IPFilterRule) IsMatchAllIPFilter() bool {
 	return false
 }
 
-func (ipfRule *IPFilterRule) decodeIpFilterPortInfo(source bool, tag string) error {
```

Comment:
Why did you remove the `return` value (`error`)? Should I understand that somehow it was unused or irrelevant?

Reply:
there was no condition where it would have returned anything else then nil